### PR TITLE
Remove unreachable! from test codes

### PR DESCRIPTION
**Pull request by NamsooCho**

### CODE REVIEW

File: src/toxcore/dht/server.rs [hunk 1]
```diff
@@ -365,11 +374,11 @@ mod tests {
         }
 
         let (pk, sk) = gen_keypair();
-        let (tx, rx) = mpsc::unbounded();
-        let alice = Server::new(tx, pk, sk);
+        let (tx, rx) = mpsc::unbounded::<(DhtPacket, SocketAddr)>();
+        let alice = Server::new(tx, pk, sk);
```
Comment (kurnevsky) [inline src/toxcore/dht/server.rs]: Remove spaces.

File: src/toxcore/dht/server.rs [hunk 2]
```diff
@@ -365,11 +374,11 @@ mod tests {
...
 
         let (pk, sk) = gen_keypair();
-        let (tx, rx) = mpsc::unbounded();
-        let alice = Server::new(tx, pk, sk);
+        let (tx, rx) = mpsc::unbounded::<(DhtPacket, SocketAddr)>();
+        let alice = Server::new(tx, pk, sk);            
         let (bob_pk, bob_sk) = gen_keypair();
         let precomp = precompute(&alice.pk, &bob_sk);
-
+
```
Comment (kurnevsky) [inline src/toxcore/dht/server.rs]: Remove spaces.

File: src/toxcore/dht/server.rs (hunk 1 above)
Comment (NamsooCho) [reply src/toxcore/dht/server.rs]: fixed

File: src/toxcore/dht/server.rs (hunk 2 above)
Comment (NamsooCho) [reply src/toxcore/dht/server.rs]: fixed

**Pull request merged**